.github/workflows/build-toolchain.yml

```diff
@@ -284,8 +299,10 @@ jobs:
             fi
           fi
 
-          echo windows_build_runner=${{ inputs.windows_default_runner || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
-          echo windows_compilers_runner=${{ inputs.windows_compilers_runner || inputs.windows_default_runner || vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_x64_build_runner=${{ inputs.windows_x64_default_runner || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_x64_compilers_runner=${{ inputs.windows_x64_compilers_runner || inputs.windows_x64_default_runner || vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_arm64_build_runner=${{ inputs.windows_arm64_default_runner || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
```

Comment:
This is not going to work as it defaults to `windows-latest`, which is an x64 host. Mismatch between the actual bot architecture and the `build_arch` input will cause issues.

Reply:
Right, agree that's not ideal! What'd happen currently as things are is that it partially works as long as we always pass in `inputs.windows_x64_default_runner` and `inputs.windows_arm64_default_runner`, which I think is currently the case as we override with the cirun machines. If you know of the name to specify the arm64 equivalent of `windows-latest`, lmk. I do not. I don't see a way around it other than requiring the runner parameters in the input and drop the default.

Reply:
Unfortunately, there is currently no public arm64 windows runner. I believe we should have a toggle for the Windows arm64 build but that definitely complicates things. Let's keep it as-is for now. I'll follow up later when the mac build is in a better state.

Reply:
Gotcha